doc/sphinx/conf.py

```diff
@@ -168,7 +168,8 @@
 #       the sphinx_reredirects extension will be disabled.
 redirects = {
     'how-to/how-to-enable-graphics-core22-on-a-device': '../how-to-enable-graphics-for-snaps-on-a-device',
-    'explanation/ok-so-what-is-this-wayland-thing-anyway': '../../how-to/developing-a-wayland-compositor-using-mir'
+    'explanation/ok-so-what-is-this-wayland-thing-anyway': '../../tutorial/write-your-first-wayland-compositor',
+    'how-to/developing-a-wayland-compositor-using-mir.md*': '../../tutorial/write-your-first-wayland-compositor'
```

Comment:
No, wildcards don't seem to work (and the `.md` would've broke it anyway.
```suggestion
    'how-to/developing-a-wayland-compositor-using-mir': '../../tutorial/write-your-first-wayland-compositor'
```